### PR TITLE
Examples of alerts

### PR DESCRIPTION
**Pull request by hweawer**

### CODE REVIEW

File: alerts/alerts.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      - alert: UnvetterBotNoMetrics
+        expr: absent_over_time(unvetter_bot_build_info{service="lido-unvetter-bot"}[5m])
+        for: 3m
+        labels:
+          severity: warning
+          app_team: tooling
+        annotations:
+          summary: "Unvetter doesn't reporting metrics for too long."
+          description: "Unvetter doesn't reporting metrics for too long."
+  - name: Transport stuck
```
Comment (F4ever) [inline alerts/alerts.yml:35]:
```suggestion
  - name: No transport heartbeat
```

File: alerts/alerts.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    rules:
+      - alert: NoMessagesReceivedFor6h
+        expr: sum by (service,address) (changes(label_replace({__name__=~"depositor_bot_ping_messages|depositor_bot_deposit_messages|pauser_bot_ping_messages|pauser_bot_pause_messages|unvetter_bot_ping_messages|unvetter_bot_unvet_messages"},"name_label","$1","__name__", "(.+)")[6h:])) == 0
```
Comment (F4ever) [inline alerts/alerts.yml:38]: Do not aggregate by `address`. This should be a common event if bot can't receive messages.

File: alerts/alerts.yml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        labels:
+          severity: warning
```
Comment (F4ever) [inline alerts/alerts.yml:40]:
```suggestion
          severity: critical
```

File: alerts/alerts.yml [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+          summary: "No activity detected for the last 6 hours for address {{ $labels.address }}"
+          description: "None of the counters (deposit, pause, unvet, ping) have increased in the last 6 hours for the address {{ $labels.address }}."
+  - name: Depositor bot. Account balance
+    rules:
+      - alert: DepositorBotAccountBalanceLow
+        expr:
+          depositor_bot_account_balance{service="lido-depositor-bot"}/10^18 < 3
+          UNLESS ON(service)
+          depositor_bot_build_info{service="lido-depositor-bot",CREATE_TRANSACTIONS="False"}
+        for: 5m
```
Comment (F4ever) [inline alerts/alerts.yml:52]: remove for

File: alerts/alerts.yml [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+          severity: warning
+          app_team: tooling
+        annotations:
+          summary: "Pauser doesn't reporting metrics for too long."
+          description: "Pauser doesn't reporting metrics for too long."
+  - name: Unvetter bot. No metrics
+    rules:
+      - alert: UnvetterBotNoMetrics
+        expr: absent_over_time(unvetter_bot_build_info{service="lido-unvetter-bot"}[5m])
+        for: 3m
```
Comment (F4ever) [inline alerts/alerts.yml:28]: remove

File: alerts/alerts.yml [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+          description: "Account balance is {{ $value }} ETH."
+
+      - alert: DepositorBotAccountBalanceCriticalLow
+        expr:
+          depositor_bot_account_balance{service="lido-depositor-bot"}/10^18 < 1
+          UNLESS ON(service)
+          depositor_bot_build_info{service="lido-depositor-bot",CREATE_TRANSACTIONS="False"}
+        for: 5m
+        labels:
+          severity: warning
```
Comment (F4ever) [inline alerts/alerts.yml:67]: major

**Comment by hweawer** [comment]
Made all of the alerts consistent with our alerts

**Review by F4ever — APPROVED** [review]

**Review by F4ever — APPROVED** [review]

**Pull request merged**